db_stress_tool/multi_ops_txns_stress.h

```diff
@@ -345,6 +345,9 @@ class MultiOpsTxnsStressTest : public StressTest {
   uint32_t GenerateNextC(ThreadState* thread);
 
 #ifndef ROCKSDB_LITE
+  void ProcessRecoveredPreparedTxnsHelper(Transaction* txn,
+                                          SharedState*) override;
```

Comment:
Do you mind reminding me what problem led to this override being needed (preferably in the form of adding a comment)?

Reply:
That's because multiops txn doesn't need to update the ExpectedState to be UNKNOWN as the base class StressTest does. Will add comment.

Reply:
Fixed